ui/js/selectors/claims.js

```diff
@@ -185,6 +185,22 @@ export const selectMyClaimsOutpoints = createSelector(
   }
 );
 
+export const selectMyClaimSdHashesByOutpoint = createSelector(
```

Comment:
Can you help me understand why this is necessary? When do I have an outpoint that I need to translated into an sd_hash? Can we just stored sd_hashes in that place instead?

Reply:
I think it is necessary due to what happens when I have a pending publish(related to https://github.com/lbryio/lbry-app/pull/735#discussion_r151756127 probably).